dask/dataframe/core.py

```diff
@@ -4123,6 +4123,12 @@ def to_timedelta(arg, unit='ns', errors='raise'):
                           meta=meta)
 
 
+@wraps(pd.isna)
+def isna(arg):
+    meta = pd.Series([True])
+    return map_partitions(pd.isna, arg, meta=meta)
```

Comment:
I suspect that if the input series has a name or other metadata (I'm not sure if other metadata exists) then this information will not be captured with the `meta=` definition above.  I suspect that you can trigger a failure in your test by adding `name='foo'` to the construction of your input series (the pandas version will be named while the dask.dataframe version will not be named)

I looked at map_partitions to see how it handles metadata and it seems to be using a function `dask.dataframe.core._emulate`.  I'm curious, can we let it handle this and not provide `meta=` as a keyword argument here?  It might do the right thing by default.

Reply:
Seems like it is working, I will try and make the test fail as well by adding the `name='foo'`.